src/gordon_gcp/clients/auth.py

```diff
@@ -178,14 +178,38 @@ async def refresh_token(self):
                 specifically a :exc:`.GCPHTTPResponseError`, if the
                 exception is associated with a response status code.
         """
+        if isinstance(
+                self.creds, compute_engine.credentials.Credentials):
+            await self._refresh_token_using_compute_credentials()
+        else:
+            await self._refresh_token_using_service_account_credentials()
+
+    async def _refresh_token_using_compute_credentials(self):
+        metadata_host = os.getenv(environment_vars.GCE_METADATA_ROOT,
+                                  'metadata.google.internal')
+        url = "http://{0}/" \
+              "computeMetadata/v1/instance/service-accounts/{1}/token".format(
+                metadata_host,
+                self.creds._service_account_email)
```

Comment:
- on this line and others `"` should be `'` (by convention, code should be consistent in its choice of single vs double quote, so we went with single, besides the case of nested quotes, for example `x = log.error('an error with "something"')` )
- Python 3 also has format strings, which can make code a little more readable. We also use parentheses for multiline strings. For example this could become (hopefully I got the whitespace right 🤞)
```py3
        url =  (f'http://{metadata_host}/computeMetadata/'
                'v1/instance/service-accounts/'
                f'{self.creds._service_account_email}/token')
```

Here's the style guide we're following for reference: https://google.github.io/styleguide/pyguide.html

Reply:
- yep, I missed to make those single quotes
- Will change to format strings